### ISSUE TITLE
Making Our Own Channel

### ISSUE THREAD

**Issue by DevilsDesigns** [n0tMaster/n0tFlix-Main#6]
Is there a way to copy a base template and add other websites IE. PlutoTV or TubeGalore.com, or 123MOVIES.COM or any other Streaming sites that have free non subscription watching features. If so can you explain how to? If not is this possible?

**Comment by TheOddPirate** [comment]
i used to have a template, but after i started school again i stoped working on the jellyfin plugins. but its time for my winter break now so im seting up my computer for some jellyfin development and if i dont loose intrest to fast il try to upload my themplates

**Comment by DevilsDesigns** [comment]
Yes please been waiting ages. I figured you were afk fit good. Ie dead. Thank you so much I can't wait.

**Comment by DevilsDesigns** [comment]
You can discord me if it'd easier. Usernane: DemonWarrior#2021

**Comment by TheOddPirate** [comment]
if you look in the template forlder u can find "n0tFlix.Channel.Name"  this is the base template i use everytime i start a new channel, after this most of the work is sniffing information on the websites u want to stream the data from.
its fun to do the first 2-3 times but after this it gets  really boring.  

Im working on getting most of the previous plugins up to date right now, but after i finish them il contact you on discord and we could help each other create all the channel jellyfin need to become the best it can, i would love to see a plutoTV plugin but i havent been able to get plutoTV working here in norway yet

**Comment by DevilsDesigns** [comment]
okay will do bud. Thank you so much!

**Comment by DevilsDesigns** [comment]
any update yet?

**Comment by TheOddPirate** [comment]
sorry its going slow, life gets in the way and im losing my motivation to fast, BUT i just uploaded a new repository with some subtitle plugins, i just need to figure out how this manifest file works now so i can check if they are working
https://github.com/n0tOdd/n0tFlix
